ServerCore/Areas/Identity/IdentityHostingStartup.cs

```diff
@@ -14,9 +15,28 @@ public void Configure(IWebHostBuilder builder)
         {
             builder.ConfigureServices((context, services) =>
             {
-                services.AddDbContext<PuzzleServerContext>(options =>
-                    options.UseSqlServer(
-                        context.Configuration.GetConnectionString("PuzzleServerContext")));
+                // Use SQL Database if in Azure, otherwise, use localdb
+                if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
+                {
+                    // Set up to use Azure settings
+                    IHostingEnvironment env = context.HostingEnvironment;
+                    IConfigurationBuilder configBuilder = new ConfigurationBuilder()
+                        .SetBasePath(env.ContentRootPath)
+                        .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+                        .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+                        .AddEnvironmentVariables();
+                    context.Configuration = configBuilder.Build();
+
+                    services.AddDbContext<PuzzleServerContext>
+                        (options => options.UseLazyLoadingProxies()
+                            .UseSqlServer(context.Configuration.GetConnectionString("PuzzleServerSQLConnectionString")));
```

Comment:
Please document where our secrets are and how to set them up as part of this so we can set up real production if we need it

Reply:
The secrets store isn't set up yet, right now we're using the Azure Application Settings - a future change will turn on KeyVault access and I'll add docs for using that.